internal/commands/download_sbom.go

```diff
@@ -0,0 +1,41 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	cpkg "github.com/buildpacks/pack/pkg/client"
+	"github.com/buildpacks/pack/pkg/logging"
+)
+
+type DownloadSBOMFlags struct {
+	Local          bool
+	Remote         bool
+	DestinationDir string
+}
+
+func DownloadSBOM(
+	logger logging.Logger,
+	client PackClient,
+) *cobra.Command {
+	var flags DownloadSBOMFlags
+	cmd := &cobra.Command{
+		Use:     "download-sbom <image-name>",
+		Args:    cobra.ExactArgs(1),
+		Short:   "Download SBoM from specified image",
+		Example: "pack download-sbom buildpacksio/pack",
```

Comment:
nit: Add a Long, maybe clarifying in it what an SBoM is?

Reply:
_Addressed in latest commit._